src/main/scala/com/linkedin/feathr/offline/source/dataloader/BatchDataLoader.scala

```diff
@@ -63,16 +63,14 @@ private[offline] class BatchDataLoader(ss: SparkSession, location: InputLocation
     val dataIOParametersWithSplitSize = Map(SparkIOUtils.SPLIT_SIZE -> inputSplitSize) ++ dataIOParameters
     val dataPath = location.getPath
 
-    log.info(s"Loading ${dataPath} as DataFrame, using parameters ${dataIOParametersWithSplitSize}")
+    log.info(s"Loading ${location} as DataFrame, using parameters ${dataIOParametersWithSplitSize}")
     try {
-      if (dataPath.startsWith("jdbc")){
-        JdbcUtils.loadDataFrame(ss, dataPath)
-      } else {
```

Comment:
@blee1234 May I ask why did you write this? InputLocation.loadDf exists since the beginning, why do you hack all the way to break its function?

Reply:
hey @windoze I actually just refactored that path from location.getPath to dataPath. 

https://github.com/linkedin/feathr/commit/3dc354969a6bc470eeae52276910cfa0fe09130d#diff-28e361a441ef37dd98fad74ea933606d35c3ad7a1beb5bf0c026438fdd8b42f3R68-R70

I don't think that refactoring is what you're referencing in your comment. The whole segment you're referencing was actually in the repo from this commit.

https://github.com/linkedin/feathr/commit/3b1a2ea089afba724ff8dba14f10b913e161149a#diff-8d2002a450fb2814a262118070cf8a9ed0564b76c27a0cd7c4360cf6480c8e5fR64-R66

where the file was originally called SparkDataLoader and then renamed later to BatchDataLoader.